hoot-core-test/src/test/cpp/hoot/core/algorithms/AlphaShapeTest.cpp

```diff
@@ -48,13 +46,13 @@
 // Tgs
 #include <tgs/Statistics/Random.h>
 
-#include "../TestUtils.h"
-
 using namespace geos::geom;
-using namespace hoot;
 using namespace std;
 
-class AlphaShapeTest : public CppUnit::TestFixture
+namespace hoot
```

Comment:
Quite a few classes weren't in the `hoot` namespace, thus requiring the `using namespace hoot` above.